minifi-toolkit/minifi-toolkit-configuration/src/test/resources/CsvToJson.yml

```diff
@@ -13,6 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+MiNiFi Config Version: 2
```

Comment:
The config files in the toolkit were updated but not the ones in minifi-bootstrap or the default one that gets bundled (in minifi-resources). Could you update those too?

Kinda sucks that we have test yml files in multiple locations that will need to be updated each time there is a change to the schema. Any thoughts to remedy this/if you think it's worth it?